src/lsp/cobol_common/srcloc.ml

```diff
@@ -273,6 +273,10 @@ let rec in_area_a: srcloc -> bool = function
   | Rpl { in_area_a; _ } -> in_area_a
   | Cat { left; _ } -> in_area_a left
 
+let is_pointwise: srcloc -> bool = function
+  | Raw (s, e, _) -> s.pos_cnum == e.pos_cnum && s.pos_fname = e.pos_fname
```

Comment:
Why physical equality ?

Reply:
Well. `cnum`s are `int`s, so it does not really matter.